knimin/lib/tests/test_data_access.py

```diff
@@ -19,10 +19,64 @@ def setUp(self):
             db.add_external_survey('Vioscreen', 'FFQ', 'http://vioscreen.com')
         except ValueError:
             pass
+        # Populate some field options
+        sql = """INSERT INTO pm.plate_type (name, cols, rows, notes)
+                 VALUES ('96-well', 12, 8, 'Standard 96-well plate')"""
+        db._execute_sql(sql)
+        sql = """INSERT INTO pm.extraction_robot (name) VALUES ('HOWE_KF1')"""
```

Comment:
You can insert multiple values in the same table by using a single INSERT command:
```sql
INSERT INTO pm.extraction_robot (name)
VALUES ('HOWE_KF1'), ('HOWE_KF2'), ('HOWE_KF3'), ('HOWE_KF4')
```